cdap-docs/application-templates/source/etl/templates/sinks/batch/snapshotavro.rst

```diff
@@ -0,0 +1,60 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+==========================
+Sinks: Batch: SnapshotAvro
+==========================
+
+.. rubric:: Description
+
+A batch sink for a FileSet that writes a snapshot of data in Avro format.
+At the end of every pipeline run, the previous run's data will be overwritten
+with the current run's data. All data for the run will be written to that
+location in the file system.
+
+.. rubric:: Use Case
+
+This sink is used whenever you want access to a FileSet containing exactly the most
+recent run's data in Avro format. Alternatively, it is used whenever you would like
+the output of a run to be written to a constant location in a file system. For example,
+you might want to create daily snapshots of a database by reading the entire contents of
+the table and writing to this sink, then have other programs analyze the contents of the
```

Comment:
a table, writing to this sink, and then other programs can analyze the contents of the specified file.